mne/time_frequency/tests/test_stockwell.py

```diff
@@ -81,6 +83,12 @@ def test_stockwell_api():
                                    return_itc=True)
         if fmax is not None:
             assert_true(power.freqs.max() <= fmax)
+        power_evoked = tfr_stockwell(epochs.average(), fmin=fmin, fmax=fmax,
+                                     return_itc=False)
+        # for multitaper these don't necessarily match, but they seem to
+        # for stockwell... if this fails, this maybe could be changed
+        # just to check the shape
```

Comment:
I can only match if all epochs are equal or don't overlap in the TF domain.

can it be the reason it works here?